net2/config.js

```diff
@@ -242,16 +242,21 @@ function reloadFeatures() {
   }
 
   for (const f in callbacks) {
-    if (!features)
+    if (!features) {
+      features = {}
```

Comment:
`features = featuresNew` here might be better, `featuresNew` might not have every entry enabled